WebJobs.Script.sln

```diff
@@ -288,13 +288,15 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "HttpTrigger-Java", "HttpTri
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "WebJobs.Script", "src\WebJobs.Script\WebJobs.Script.csproj", "{BCFBE330-871A-44BE-9016-83845231230F}"
 EndProject
-Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "WebJobs.Script.Host", "src\WebJobs.Script.Host\WebJobs.Script.Host.csproj", "{180EF900-AF1F-4EAE-8EF0-50A3763BDE15}"
```

Comment:
Why is this being removed?

Reply:
This is not as needed with the new model. If the goal is to have a sample, I think we should probably have a richer one outside of the solution, but to run the application as a console app is something we can do with the WebHost. Thoughts?

Reply:
I use this sample and I'd like to retain it. It should be easy to retain this capability, right? It's a good forcing function for layering as well. For all 3 repos, I've added a simple sample host like this.